app/components/forms/wizard/other-details-step.js

```diff
@@ -23,39 +23,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return orderBy(licenses, 'name');
   }),
 
-
   socialLinks: computed('data.event.socialLinks.@each.isDeleted', function() {
     return this.data.event.socialLinks.filterBy('isDeleted', false);
   }),
 
   isUserUnverified: computed('authManager.currentUser.isVerified', function() {
     return !this.authManager.currentUser.isVerified;
   }),
-  /**
-   * returns the validation rules for the social links.
-   */
-  socialLinksValidationRules: computed('socialLinks', function() {
-    let validationRules = {};
-    for (let i = 0; i < this.socialLinks.length; i++) {
-      validationRules = merge(validationRules, {
-        [this.socialLinks.get(i).identifier]: {
-          identifier : this.socialLinks.get(i).identifier,
-          optional   : true,
-          rules      : [
-            {
-              type   : 'regExp',
-              value  : protocolLessValidUrlPattern,
-              prompt : this.l10n.t('Please enter a valid url')
-            }
-          ]
-        }
-      });
-    }
-
-    return validationRules;
-  }),
-
-
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6103967)